sdks/java/io/elasticsearch-tests/elasticsearch-tests-5/src/test/java/org/apache/beam/sdk/io/elasticsearch/ElasticsearchIOTest.java

```diff
@@ -202,4 +202,10 @@ public void testWriteRetry() throws Throwable {
     elasticsearchIOTestCommon.setPipeline(pipeline);
     elasticsearchIOTestCommon.testWriteRetry();
   }
+
+  @Test
+  public void testWriteRetryValidRequest() throws Throwable {
```

Comment:
Should this be added to the the ES2 and ES6 tests as well?

Reply:
done